src/lib.rs

```diff
@@ -179,7 +179,8 @@ impl AndroidLogger {
 
 static ANDROID_LOGGER: OnceLock<AndroidLogger> = OnceLock::new();
 
-const LOGGING_TAG_MAX_LEN: usize = 23;
+// Maximum length of a tag that does not require allocation.
+const LOGGING_TAG_MAX_LEN: usize = 127;
```

Comment:
Nice opportunity for a doc-comment and a note regarding the NUL terminator being exempt from this limit?

```suggestion
/// Maximum length(excluding NUL terminator) of a tag that does not require allocation.
const LOGGING_TAG_MAX_LEN: usize = 127;
```